server/gitrest/packages/gitrest-base/src/utils/isomorphicgitManager.ts

```diff
@@ -276,7 +276,14 @@ export class IsomorphicGitRepositoryManager implements IRepositoryManager {
             ]);
             return conversions.refToIRef(resolvedRef, expandedRef);
         } catch (err) {
-            winston.error(`getRef error: ${safeStringify(err, undefined, 2)} repo: ${this.repoName} ref: ${refId}`);
+            Lumberjack.error(
+                "getRef error",
+                {
+                    ...this.lumberjackBaseProperties,
+                    repo: this.repoName,
```

Comment:
I voerlooked this since I was just reusing the same names in the original winston logs. We could centralize all of the properties in server-services-telemetry, as we do with other properties like tenantId and documentId, but that would require another pre-release. I will instead put those properties in a central place in GitRest. And then later we can move them to server-services-telemetry if we think that makes sense.

As for repoName and repoOwner, I understand they might become redundant later, but I think it does not hurt to log those properties for now. The extra cost of having those 2 extra properties in the "properties" column in the logs is likely negligible.

Reply:
Oohh I totally forgot they are all lumped into the "properties" column, not their own column. Thanks!